core/src/main/java/de/danoeh/antennapod/core/glide/ApOkHttpUrlLoader.java

```diff
@@ -111,10 +109,16 @@ public boolean handles(@NonNull String s) {
 
         @Override
         public Response intercept(Chain chain) throws IOException {
-            if (NetworkUtils.isDownloadAllowed()) {
+            if (NetworkUtils.isImageAllowed()) {
                 return chain.proceed(chain.request());
             } else {
-                return null;
+                return new Response.Builder()
+                        .code(420)
```

Comment:
huh?

Reply:
Since Glide was upgraded, AntennaPod crashes when it reaches this code path (NPE). (Because I previously used `append` instead of `replace` when setting up the OkHttpLoader, this code path was not entered a lot - but the app also did not respect the mobile updates preference). There must be a response with a code to make Glide happy, so I scrolled through the http response codes and found `420: Policy Not Fulfilled` to be a good candidate. The app seems to work properly with every code that is not 2xx